coalib/parsing/DefaultArgParser.py

```diff
@@ -49,7 +41,7 @@ def default_arg_parser(formatter_class=None):
 <https://github.com/coala/bear-docs/blob/master/README.rst#supported-languages>
 or run:
 
-    $ coala --show-bears --filter-by-language C Python
+    $ coala-show --bears --filter-by-language C Python
```

Comment:
I've done some work and changed the doc, but then decided to do that in a different issue, so i have to change it back :)